src/scripts/visualizations/tooltip.js

```diff
@@ -9,7 +9,8 @@
             distance: 5,
             distanceX: undefined,
             distanceY: undefined,
```

Comment:
Would it make sense to say distance can be a number or an object of the form {x :<x>, y: <y>}? That way we'll never have overlapping contradictory values